Feature 1

sabatjohn17 opened the pull request.

sabatjohn17 committed:
Add information about Pakistan and Bengal tiger

- Add content to pakistan.txt with basic country information

- Create new file tiger.txt with Bengal tiger facts

sabatjohn17 committed:
Fix typo and add population info for Pakistan

- Correct typo in country name (kistan -> Pakistan)

- Add information about Pakistan's population ranking and Muslim population

ajit-spec reviewed (changes requested):
please add 1line in india.txt file as well

sabatjohn17 committed:
Add historical information to India description

- Include fact about India being home to one of the world's oldest civilizations

- Add detail about 5,000 years of history

ajit-spec reviewed (approved):
good
